history-of-the-quakers/original/31-book-3-chapter-5.adoc

```diff
@@ -113,11 +118,16 @@ and the other some time after, a copy of each whereof I insert,
 as a specimen of the honest plain-dealing of men, who, with Elihu, knew not to flatter,
 lest in so doing their maker should take them away.
 
-"`To the KING.
+[.embedded-content-document.letter]
```

Comment:
There are two embedded documents here that are called epistles in the text, but i labeled as letters because i didn't think they were true epistles, but idk!

Reply:
ok, i think you made the right call 👍